### PR TITLE
Remove Litmus in favour of Beaker

### PR DESCRIPTION
**Pull request by dhoppe**

### CODE REVIEW

**Comment by dhoppe** [comment]
This seems to be an issue with the Docker images:
```
Apr 27 10:14:09 centos8-64-1 systemd[1]: chronyd.service: Refusing to accept PID outside of service control group, acquired through unsafe symlink chain: /run/chrony/chronyd.pid
```

**Comment by ekohl** [comment]
@dhoppe isn't that the issue we've been seeing where `PIDFile` is used in the systemd service?

File: manifests/config.pp [hunk 1]
```diff
@@ -3,34 +3,41 @@
...
+  $local_stratum        = $chrony::local_stratum,
   $log_options          = $chrony::log_options,
   $refclocks            = $chrony::refclocks,
-  $mailonchange         = $chrony::mailonchange,
-  $threshold            = $chrony::threshold,
-  $lock_all             = $chrony::lock_all,
   $peers                = $chrony::peers,
   $servers              = $chrony::servers,
   $pools                = $chrony::pools,
+  $makestep_seconds     = $chrony::makestep_seconds,
```
Comment (ekohl) [inline manifests/config.pp:25]: This looks like an unrelated change.

**Comment by alexjfisher** [comment]
Why are you getting rid of litmus?  It was working fine in this module.

File: manifests/config.pp (hunk 1 above)
Comment (dhoppe) [reply manifests/config.pp:25]: This parameter is required by the templates.

**Comment by dhoppe** [comment]
Unfortunately I did a modulesync and did not know our position related to Beaker vs Litmus.
